prow/test/integration/integration-test.sh

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env bash
+# Copyright 2021 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+bazel=$(command -v bazelisk || command -v bazel)
+
+retry() {
```

Comment:
This is so much simpler, hardcoded 3 in the function. Prefer to make it a function to make main cleaner. Also prefer to keep the "... retrying..." log to avoid confusions.